soc/snps/nsim/arc_v/rmx5xx/CMakeLists.txt

```diff
@@ -0,0 +1,17 @@
+# SPDX-License-Identifier: Apache-2.0
```

Comment:
For classic, there is another hierarchy for the series (em/hs/vpx). Shouldn't we already do that for ARC-V as well (rmx/rhx/rpx)?